src/interface/obsidian/src/settings.ts

```diff
@@ -141,9 +189,8 @@ export class KhojSettingTab extends PluginSettingTab {
                     this.plugin.registerInterval(progress_indicator);
 
                     this.plugin.settings.lastSync = await updateContentIndex(
-                        this.app.vault, this.plugin.settings, this.plugin.settings.lastSync, true
+                        this.app.vault, this.plugin.settings, this.plugin.settings.lastSync, true, true
                     );
-                    new Notice('✅ Updated Khoj index.');
```

Comment:
Should we leave this notice here? Can be a helpful indicator to the user.

Reply:
Yeah, it's been moved into the index updater function. We now only show the indexing success notice when manually triggered by user and indexing succeeded. Previously we'd show indexing success notice even on error, after the error notice which was weird